packages/instant-apps-components/src/components/instant-apps-header/instant-apps-header.test.ts

```diff
@@ -0,0 +1,66 @@
+import { expect, test } from 'vitest';
+import '../../../dist/components/instant-apps-header.js';
+const header = document.createElement('instant-apps-header');
+header.setAttribute('id', 'instant-apps-header');
+header.setAttribute('background-color', '#151515');
+test('should render component with custom title text', async () => {
```

Comment:
This applies to the whole PR, but we may need to think about how to format our tests, i.e using `describe` for a whole test suite and `test` for individual tests